src/test/amazonqFeatureDev/controllers/chat/controller.test.ts

```diff
@@ -8,41 +8,42 @@ import * as assert from 'assert'
 import * as path from 'path'
 import sinon from 'sinon'
 import { waitUntil } from '../../../../shared/utilities/timeoutUtils'
-import { ControllerSetup, createController } from '../../utils'
-import { ChatControllerEventEmitters } from '../../../../amazonqFeatureDev/controllers/chat/controller'
+import { ControllerSetup, createController, createSession } from '../../utils'
 import { FollowUpTypes, createUri } from '../../../../amazonqFeatureDev/types'
 import { Session } from '../../../../amazonqFeatureDev/session/session'
 import { Prompter } from '../../../../shared/ui/prompter'
-import { toFile } from '../../../testUtil'
+import { assertTelemetryCurried, toFile } from '../../../testUtil'
 import { SelectedFolderNotInWorkspaceFolderError } from '../../../../amazonqFeatureDev/errors'
+import { PrepareRefinementState } from '../../../../amazonqFeatureDev/session/sessionState'
+import { FeatureDevClient } from '../../../../amazonqFeatureDev/client/featureDev'
 
 describe('Controller', () => {
     const tabID = '123'
     const conversationID = '456'
     const uploadID = '789'
 
+    let session: Session
     let controllerSetup: ControllerSetup
 
+    before(() => {
+        sinon.stub(performance, 'now').returns(0)
+    })
+
     beforeEach(async () => {
-        controllerSetup = await createController({
-            conversationID,
-            tabID,
-            uploadID,
-        })
+        controllerSetup = await createController()
+        session = await createSession({ messenger: controllerSetup.messenger, conversationID, tabID, uploadID })
     })
 
     afterEach(() => {
         sinon.restore()
     })
 
     describe('openDiff', async () => {
-        async function openDiff(
-            controllerEventEmitter: ChatControllerEventEmitters,
-            filePath: string,
-            deleted = false
-        ) {
+        const reviewedChangesTelemetry = assertTelemetryCurried('amazonq_isReviewedChanges')
```

Comment:
does this mean we can remove `amazonq_isReviewedChanges` from the exceptions list in https://github.com/aws/aws-toolkit-vscode/blob/170588fcf4b6fb7d2a6eda7eed17262fbeb049f8/src/shared/telemetry/exemptMetrics.ts#L17-L19